lms/static/js/student_account/register.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Registration template JS.
+ */
+
+(function($, gettext) {
+    'use strict';
+    var buildModal = function(link, modal_class, content_class, close_button_class) {
+        // create the modal container with a title, close button and content area
+        var close_label = gettext('Close'),
+            title_id = 'modal-header-text',
+            modal = $('<div>', {
+                'class': modal_class
+            }),
+            content = $('<div>', {
+                'class': content_class,
+                role: 'dialog',
```

Comment:
I used quotes around `class` because it's a keyword. `role` is not so it can be used safely. The `aria-*` ones needs them because the dash is also an operator. I added the quotes anyway for consistency :)